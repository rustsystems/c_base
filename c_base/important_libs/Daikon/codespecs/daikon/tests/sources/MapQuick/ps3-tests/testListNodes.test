#test listing the nodes
CreateNode n1 2
CreateNode n2 20
CreateNode n3 4
CreateNode n4 4
CreateNode n5 12
CreateNode n6 1
CreateNode n7 3
CreateNode n8 9
CreateNode n9 5
CreateNode n10 3
CreateNode n11 11
CreateGraph A
ListNodes A
AddNode A n1
AddNode A n2
AddNode A n3
AddNode A n4
AddNode A n5
AddNode A n6
AddNode A n7
AddNode A n8
AddNode A n9
ListNodes A
