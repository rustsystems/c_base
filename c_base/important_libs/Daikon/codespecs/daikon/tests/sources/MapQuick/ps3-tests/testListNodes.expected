#test listing the nodes
created node n1 with cost 2
created node n2 with cost 20
created node n3 with cost 4
created node n4 with cost 4
created node n5 with cost 12
created node n6 with cost 1
created node n7 with cost 3
created node n8 with cost 9
created node n9 with cost 5
created node n10 with cost 3
created node n11 with cost 11
created graph A
A contains: 
added node n1 to A
added node n2 to A
added node n3 to A
added node n4 to A
added node n5 to A
added node n6 to A
added node n7 to A
added node n8 to A
added node n9 to A
A contains: n1 n2 n3 n4 n5 n6 n7 n8 n9 
